src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -188,7 +204,7 @@ class ActivityFeed extends React.Component<Props, State> {
                     <ApprovalCommentForm
                         onSubmit={() => {
```

Comment:
Do we have a preference to avoid anonymous functions in render methods?

Reply:
Ideally yes. This one can be avoided because it doesn't rely on any params?

Reply:
addressed